src/Components/WebAssembly/WebAssembly/src/targets/Microsoft.AspNetCore.Components.WebAssembly.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
     <BlazorWebAssemblyJSPath>$(MSBuildThisFileDirectory)blazor.webassembly.js</BlazorWebAssemblyJSPath>
+    <WasmEnableLegacyJsInterop Condition="'$(WasmEnableLegacyJsInterop)' == ''" >false</WasmEnableLegacyJsInterop>
```

Comment:
@maraf is this right place to set a default for native relink with workload ?

Reply:
The right place in SDK https://github.com/dotnet/sdk/blob/6ad1cd1de740d3332af31bd57afef0070b7a3329/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets#L55

Reply:
Also https://github.com/dotnet/runtime/blob/6b6b0402953526c6b81c95c2f1cf1450fbaf036a/src/mono/wasm/build/BrowserWasmApp.targets#L68

Reply:
But we should set the default value on our side